src/index.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import App from './components/app/app';
+import {Setting} from './const';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 
 root.render(
   <React.StrictMode>
-    <h1>Hello, World!</h1>
+    <App
+      cardsCount = {Setting.CardCount}
```

Comment:
```suggestion
      cardsCount={Setting.CardCount}
```
Форматируем так, ошибки линтера следует убрать